csharp/src/Ice/OutgoingResponseFrame.cs

```diff
@@ -23,35 +24,39 @@ public sealed class OutgoingResponseFrame
         /// <summary>The Ice protocol of this frame.</summary>
         public Protocol Protocol { get; }
 
-        /// <summary>The frame reply status <see cref="ReplyStatus"/>.</summary>
-        public ReplyStatus ReplyStatus => (ReplyStatus) Data[0][0];
+        /// <summary>The reply status; see <see cref="ZeroC.Ice.ReplyStatus"/>.</summary>
+        public ReplyStatus ReplyStatus { get; }
```

Comment:
Same as for IncomingResponseFrame, I'm not sure it's really useful to expose this "encoding detail" in the API and having a property for this might not be necessary given it's only useful for the exception constructor.

Reply:
Fixed.